src/firewall/core/io/firewalld_conf.py

```diff
@@ -247,6 +249,20 @@ def read(self):
                 )
             self.set("FlushAllOnReload", str(config.FALLBACK_FLUSH_ALL_ON_RELOAD))
 
+        value = self.get("ReloadPolicy")
+        try:
+            parsed = self._parse_reload_policy(value)
```

Comment:
assigning to parsed is not needed. We don't use the value. Bare `self._parse_reload_policy(value)` is sufficient.

Or even:
```python
value2 = self._unparse_reload_policy(self._parse_reload_policy(value))
```

Reply:
in the current code the variable `parsed` is used (in `else:`).